languages/rust/exercises/concept/numbers/.docs/hints.md

```diff
@@ -0,0 +1,12 @@
+## General
+- [cheats.rs - Basic Types](https://cheats.rs/#basic-types)
+- [Rust book - Scalar types](https://doc.rust-lang.org/stable/book/ch03-02-data-types.html?highlight=primitive#scalar-types)
+- [Rust reference - Integer literals](https://doc.rust-lang.org/stable/reference/tokens.html#integer-literals)
+- [Rust reference - Floating point literals](https://doc.rust-lang.org/stable/reference/tokens.html#floating-point-literals)
+
+## Calculate the production rate per second
+- Determining the success rate can be done through a [conditional statement](https://doc.rust-lang.org/stable/book/ch03-05-control-flow.html#if-expressions) or with [pattern matching](https://doc.rust-lang.org/stable/book/ch18-01-all-the-places-for-patterns.html#match-arms).
+- As Rust only allows multiplication between values of the same type, some [casting](https://doc.rust-lang.org/rust-by-example/types/cast.html) will have to be done.
```

Comment:
These are great hints. I'm wondering if we should use the term "type casting" throughout instead of just "casting". e.g. to make it more explicit and possibly consistent with discussions in other languages and learning materials.